docs/2_1_common_api.adoc

```diff
@@ -500,9 +500,9 @@ All elements of the array, including structural zeros, are serialized.
 _[Example: A 2D matrix_
 [latexmath]
 ++++
-A = \left( \begin{array}{cc} a_{11}&a_{12}\\
-                             a_{21}&a_{22}\\
-                             a_{31}&a_{32}\\
+A = \left( \begin{array}{cc} a_{\mathit{11}}&a_{\mathit{12}}\\
```

Comment:
I think there's no need to use \mathit on numbers. It's only on text, as that's where latex does funny stuff.
This comment applies to everywhere else where changes were made.

Reply:
fixed